equinix/resource_fabric_routing_protocols_acc_test.go

```diff
@@ -0,0 +1,105 @@
+package equinix
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	v4 "github.com/equinix-labs/fabric-go/fabric/v4"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+)
+
+func TestAccFabricCreateRoutingProtocols(t *testing.T) {
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: checkRoutingProtocolsDelete,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccFabricCreateRoutingProtocolsConfig(7, 3),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckTypeSetElemNestedAttrs("equinix_fabric_routing_protocols.test", "direct_routing_protocol.direct_ipv4.*", map[string]string{
+						"equinix_iface_ip": fmt.Sprintf("190.1.1.1/29"),
+					}),
+				),
+				ExpectNonEmptyPlan: true,
```

Comment:
It would be helpful to test imports to ensure we aren't drifting in state by executing a refresh using an import of the identifier.

```suggestion
				ExpectNonEmptyPlan: true,
				ImportState:             true,
				ImportStateVerify:       true,
				
```

Reply:
Assuming we include Import in the resource schema, which we should be doing for every resource.

Reply:
Agree, change made.